crypto/evp/evp_enc.c

```diff
@@ -28,6 +28,15 @@
 #include "crypto/evp.h"
 #include "evp_local.h"
 
+#if defined(__GNUC__) || defined(__clang__)
+    #define likely(x) __builtin_expect(!!(x), 1)
+    #define unlikely(x) __builtin_expect(!!(x), 0)
+#else
+    #define likely(x) x
+    #define unlikely(x) x
+#endif
+
+
```

Comment:
It would make sense to put this into some header file in include/internal (for example `include/internal/common.h`) so it can be reused elsewhere. Could you please move it?

Reply:
OK, fixed.